visgraph/graphcore.py

```diff
@@ -95,6 +97,25 @@ def fromJson(self, graph):
             node2 = (n2, graph['nodes'][n2])
             self.addEdge(node1, node2, eid=eid, eprops=eprops)
 
+    def clone(self, graph):
```

Comment:
fair enough.  idea is to populate the graph with another graph.  perhaps the name "populate()" is better.